### PR TITLE
Azure listener docs

### PR DESCRIPTION
**Pull request by tzhukov-algo**
Just added the docs for the Azure listener, with corrections @StephanieKim added in google docs.

### CODE REVIEW

File: _pages/integrations/azure_listeners.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+layout: article
+title: "Event Listeners via Azure SB"
+excerpt-short: "Set up an Event Listener to run an Algorithm with inputs from an Azure Service Bus Queue"
+categories: [integrations]
+tags: [integrations]
+show_related: true
+---
+
+These directions will help you to set up an Algorithmia Event Listener, which will run an algorithm with input payloads provided in an Azure Service Bus Queue. For more info about Azure Service Bus read https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: Can you change the prose to make the link inline?

```md
[Read more about Azure Service Bus.](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues)
```

File: _pages/integrations/azure_listeners.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+Contact [support@algorithmia.com](mailto:support@algorithmia.com) to obtain the following, which you will need during CloudFormation setup:
+
+* Azure Resource Manager(ARM) template for Service Bus Namespace and Queue
+* Custom Role definition file - QueueReceiver
+* Algorithmia Azure Account
+
+## 2. Invite Algorithmia account as guest account in your account’s Active Directory
+
+Using the Custom Role definition file you have gotten in step 1, create the Custom Role in your account.
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: s/you have gotten/you received/

File: _pages/integrations/azure_listeners.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+* Azure Resource Manager(ARM) template for Service Bus Namespace and Queue
+* Custom Role definition file - QueueReceiver
+* Algorithmia Azure Account
+
+## 2. Invite Algorithmia account as guest account in your account’s Active Directory
+
+Using the Custom Role definition file you have gotten in step 1, create the Custom Role in your account. 
+
+* Authenticate with Azure CLI
+* Run “$ az role definition create --role-definition GuestRole.json”
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: Remove the double quotes and replace with backticks. Let's also remove the `$` shell so it's friendly with copy/paste.

```md
Run `az role definition create --role-definition GuestRole.json`
```

File: _pages/integrations/azure_listeners.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+More details on how to do this with Azure Cli can be found here: https://docs.microsoft.com/en-us/azure/role-based-access-control/custom-roles-cli
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: s/can be found here/can be found at/
s/Cli/CLI/

File: _pages/integrations/azure_listeners.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+It can be also done via PowerShell and REST api.
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: s/api/API/

File: _pages/integrations/azure_listeners.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+* * Directory role: Use the role you created in step 2 (QueueReceiver)
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/eventlisteners/image_34.png">
+* * Select Invite to automatically send the invitation to the guest user. A notification appears in the upper right with the message Successfully invited user. The user account is now added to the directory as a guest account with the custom role.
+
+## 4. Create Service Bus Namespace and Queue 
+
+* Import ARM template and create your Service Bus Namespace and Queue.
+Info about ARM templates: https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview
+Info about Service Bus: https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#:~:text=In%20the%20left%20navigation%20pane,if%20the%20name%20is%20available
+* Create a new Template, and load the contents of the ARM template file from step 1 into the ARM template tab. Save the template and click “Deploy Template”
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: Add period to end of sentence.

File: _pages/integrations/azure_listeners.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+* Fill in the Namespace Name, Queue Name, and choose region appropriate for you. 
+
+Note: If the deployment fails, chances are there is already a namespace created with that name in the region you chose. If that is the error message you get on deployment error, you would need to redeploy with a different name until it works.
+
+## 5. Test Event Listener
+
+* Create a new Python 3.X Algorithm.
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: s/3.X/3.x/

File: _pages/integrations/azure_listeners.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/eventlisteners/image_35.png">
+
+* In web IDE replace source with the following:
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: s/In web IDE/In the web IDE/

File: _pages/integrations/azure_listeners.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+    Algorithmia.client().file("data://<username>/event_output_directory/" + input.get("filename")).put(input.get("data"))
+
+{% endhighlight %}
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/eventlisteners/image_36.png">
+* Click Save, then Build, then Publish buttons. On the algorithm page copy the full “path” of the algorithm.
+* Create Hosted Data directory “event_output_directory”
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/eventlisteners/image_37.png">
+* Create event listener -> Azure, algorithm name, algorithm version
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/eventlisteners/image_38.png">
+* To test event listener, create a new Python 3.x algorithm just like before, and replace source code with the following:
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: s/replace source code/replace the source code/

File: _pages/integrations/azure_listeners.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+
+* Add these dependencies to the created algorithm:
+
+{% highlight python %}
+azure-common==1.1.25   
+azure-servicebus==0.50.2
+{% endhighlight %}
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/eventlisteners/image_39.png">
+* Run the algorithm that would send messages
+* Check the event_output_directory, it should have a file with filename “test”
```
Comment (eliperelman) [inline _pages/integrations/azure_listeners.md]: Let's make this once sentence:

```md
* Check the `event_output_directory` for a file named “test”.
```

**Review by eliperelman — CHANGES_REQUESTED** [review]
Looks good, just have some prose updates.

**Review by stevielkim — APPROVED** [review]
Already provided feedback on google doc version so approving merge without comments

**Review by eliperelman — APPROVED** [review]

**Pull request merged**